git-hubflow-completion.bash

```diff
@@ -47,6 +47,41 @@
 # 
 # Distributed under the [MIT License](http://creativecommons.org/licenses/MIT/)
 
+__get_config ()
+{
+	local escaped_branch=$(echo $1 | sed -e 's/\./\\./g');
+	git config --get "branch.$escaped_branch.$2" 2>/dev/null
+}
+
+__git_ps1_diff_base ()
```

Comment:
As a concept this is outside the initial scope of the file, but its useful enough that I'm kinda "cool, let's put it in"